contrib/fsl_integration/sct_plugin.py

```diff
@@ -227,33 +285,496 @@ def onButtonGM(self, event):
+    DESCRIPTION = """This tool automatically compute the diffusion maps of the spinal cord. 
+    For more information, please refer to the article below.<br><br>
+    <b>Specific citation</b>:<br>
+    E Garyfallidis, M Brett, B Amirbekian, et al.
+    <i>Dipy, a library for the analysis of diffusion MRI data.
+    (2014)</i>. Front Neuroinform.
+
+    """
+
+    def __init__(self, parent):
+        super(TabPanelCompDTI, self).__init__(parent=parent,
+                                            id_=wx.ID_ANY)
+        button_gm = wx.Button(self, id=wx.ID_ANY,
+                              label="Compute DTI")
+        button_gm.Bind(wx.EVT_BUTTON, self.onButtonCDTI)
+        sizer = wx.BoxSizer(wx.VERTICAL)
+        sizer.Add(button_gm, 0, wx.ALL, 5)
+        self.sizer_h.Add(sizer)
+        self.SetSizerAndFit(self.sizer_h)
+
+    def onButtonCDTI(self, event):
+        selected_overlay = displayCtx.getSelectedOverlay()
+        filename_path = selected_overlay.dataSource
+
+        path_name = os.path.dirname(filename_path)
+        base_name = os.path.basename(filename_path)
+
+        fnamecrop, fext = base_name.split(os.extsep, 1)
+
+        fname = fnamecrop.split('_')[0]
+
+        bvec_name = "{}_bvecs.txt".format(fname)
+        bvecname_path = os.path.join(path_name, bvec_name)
+
+        bval_name = "{}_bvals.txt".format(fname)
+        bvalname_path = os.path.join(path_name, bval_name)
+
+        fa_name = "dti_FA.nii.gz"
+        faname_path = os.path.join(os.getcwd(), fa_name)
+
+        ad_name = "dti_AD.nii.gz"
+        adname_path = os.path.join(os.getcwd(), ad_name)
+
+        md_name = "dti_MD.nii.gz"
+        mdname_path = os.path.join(os.getcwd(), md_name)
+
+        rd_name = "dti_RD.nii.gz"
+        rdname_path = os.path.join(os.getcwd(), rd_name)
+
+        cmd_line = "sct_dmri_compute_dti -i {} -bval {} -bvec {}".format(filename_path, bvalname_path, bvecname_path)
+        print('Command:', cmd_line)
+        self.call_sct_command(cmd_line)
+
+        image = Image(faname_path)
+        overlayList.append(image)
+        del overlayList[0]
+        opts = displayCtx.getOpts(image)
+        opts.cmap = 'gray'
+
+        image = Image(adname_path)
+        overlayList.append(image)
+        opts = displayCtx.getOpts(image)
+        opts.cmap = 'gray'
+
+        image = Image(mdname_path)
+        overlayList.append(image)
+        opts = displayCtx.getOpts(image)
+        opts.cmap = 'gray'
+
+        image = Image(rdname_path)
+        overlayList.append(image)
+        opts = displayCtx.getOpts(image)
+        opts.cmap = 'gray'
+
+#Registraction between subject imaging (T1w or T2w) with PAM50 template
+class TabPanelREG(SCTPanel):
+    DESCRIPTION = """This tool automatically performs
+    the registration between your subject and the PAM50 template. 
+    For more information, please refer to the article below.<br><br>
+    <b>Specific citation</b>:<br>
+    B De Lenner, VS Fonov, D Louis Collins, et al.
```

Comment:
actually for all citations, just use the first author, like for references [here](https://github.com/neuropoly/spinalcordtoolbox#references)